managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3202,8 +3203,9 @@ public PositionImpl getPreviousPosition(PositionImpl position) {
             return PositionImpl.get(position.getLedgerId(), position.getEntryId() - 1);
         }
 
+        final ConcurrentNavigableMap<Long, LedgerInfo> ledgersCopied = new ConcurrentSkipListMap<>(ledgers);
```

Comment:
Since `ledgersCopied` is only a local variable, it's not necessary to use a concurrent data structure.